metadata_service/api/column.py

```diff
@@ -13,24 +15,22 @@ class ColumnDescriptionAPI(Resource):
     """
     def __init__(self) -> None:
         self.client = get_proxy_client()
-
-        self.parser = reqparse.RequestParser()
-        self.parser.add_argument('description', type=str, location='json')
-
         super(ColumnDescriptionAPI, self).__init__()
 
     def put(self,
             table_uri: str,
-            column_name: str,
-            description_val: str) -> Iterable[Union[dict, tuple, int, None]]:
+            column_name: str) -> Iterable[Union[dict, tuple, int, None]]:
         """
-        Updates column description
+        Updates column description (passed as a request body)
+        :param table_uri:
+        :param column_name:
+        :return:
         """
         try:
+            description = json.loads(request.json).get('description')
```

Comment:
What do you all think about having a check for what happens if the description is `None`? Does `self.client.put_column_description` end up handling this, or will the description be deleted?

Reply:
I've checked it locally (via Postman):
1) {"description": ""} - this request payload will truncate description
2) request payload with no "description" tag ends with {"msg": "Encountered exception: None"} and description data stay untouched

Reply:
Thanks for confirming the behavior for empty string and `None`. They behave as we expect, so no changes would be needed.